src/kmers.nim

```diff
@@ -278,6 +284,20 @@ proc haskmer*(target: pot_t; query: Bin): bool =
         return true
     return false
 
+## Counts the number of shared unique kmers
+## @param pot_t * - a pointer to a pot_t
+## @param pot_t * - a pointer to a pot_t
+## @return int - number of shared kmers
+#
+proc uniqueShared*(a,b: pot_t): int =
+ make_searchable(a)
```

Comment:
~~Do we really need to make *both* searchable? We're only iterating thru `a`, right?~~

Oh, I see. They're not sorted, so cannot simply skip repeats. So we need both to be "searchable". Ok.

Reply:
On thing I'm considering is that we don't use the search function, therefore, we don't care if the seeds are sorted. For other applications it might be useful to have sorted Khmers.